terraform/aws/scripts/install.sh.tpl

```diff
@@ -31,11 +29,11 @@ script
   fi
 
   exec /usr/local/bin/authenticator \
-    --config="/usr/local/etc/" \
+    --config="/usr/local/etc/approzium.config" \
```

Comment:
In 0.2.0, you give the full name of the file rather than a directory containing a `approzium.config` file.